config/default_records/pipeline_request_information_types/002_element_aviti_pipeline_request_information_types.wip.yml

```diff
@@ -1,8 +1,3 @@
----
-FlowcellRequestedInformationTypeForElementAviti:
-  pipeline_name: Element Aviti
-  request_information_type_key: requested_flowcell_type
-
```

Comment:
request_information_type record is explicitly defined in the default_records.yml per pipeline. I think we need this record definition here. Could you check it please?

Reply:
Thanks for adding the requested_flowcell_type and read_length requested_information_type and Element Aviti pipeline join records.